docs/source/ephys_at_swc/getting_started.md

```diff
@@ -0,0 +1,133 @@
+# Getting Started
+
+Getting started with extracellular electrophysiology can be intimidating!
+With numerous acquisition setups, software packages, preprocessing, postprocessing,
+and analysis steps to consider, it's understandable to feel daunted.
+
+A good first step in getting started is to ask for advice. Every setup
+and experiment is different, and it's often most effective to
+seek guidance early on. See the [Community](community)
+sections for details on where to get help.
+
+Before diving into the analysis, it can be useful to get an understanding
+for the history and landscape of extracellular electrophysiology.
+See the resources section for [general introductions](general-introduction))
+as well as more technical reading for deeper background.
+
+To get started with pipeline building, the
+[SpikeInterface](spikeinterface) resources are a good starting point.
```

Comment:
This assumes the reader has any idea what SI is. Would it be useful to explain what SI is, and why we're recommending it?